charts/incubator/notifiarr/questions.yaml

```diff
@@ -0,0 +1,113 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{workload}
+# Include{workloadDeployment}
+
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
+                                - variable: env
+                                  group: "App Configuration"
+                                  label: "Image Environment"
+                                  schema:
+                                    additional_attrs: true
+                                    type: dict
+                                    attrs:
+                                      - variable: DN_API_KEY
+                                        label: "DN_API_KEY"
+                                        description: "Notifiarr All Integrations API key from https://notifiarr.com"
+                                        schema:
+                                          type: string
+                                          default: "CHANGEME"
+                                      - variable: DN_UPSTREAMS_0
+                                        label: "DN_UPSTREAMS_0"
+                                        description: "Controls which IP is written to the web server log file (requestor or x-forwarded-for header). Also allows the x-webauth-user header for proxy authentication. Separate with spaces. The default kubernetes network is pre-filled."
+                                        schema:
+                                          type: string
+                                          default: "172.16.0.0/16"
+                                      - variable: DN_UI_PASSWORD
```

Comment:
maybe do 
credentials:
  username:
  password:
  
 and join them with helm?

Reply:
Just pushed a commit which I'm 90% sure does this.